#goal definition
sensor_msgs/PointCloud2 tgt_cloud
sensor_msgs/PointCloud2 cloud_plane
---
#result definition
bool success
---
#feedback
bool object_moved
